### DISCUSSION TITLE
Show existing and new connections on an interface

### DISCUSSION THREAD

**Discussion by Projects0987** [netbox-community/netbox#14471] [General]
Is there anyway to have another line below an interface similar to a virtual interface but have the ability to crate a connection to another device.  For example there is an existing connection between Device A - Device B and now there is plans to move the connection from Device A to Device C, so the planed connection will be Device C - Device B?

**Comment by candlerb** [comment]
I'm afraid not: each interface can only accept one cable termination.

You can make a cable which goes from device A to device B *and* device C, but that's probably not what you want. In any case, if there's an existing cable going into device C (which you're planning to remove to make way for the new connection), that would block it.

A similar issue is if you are planning to move a device to a new location. You cannot give the device both a "current" and "planned" location; nor can the location of a "planned" device overlap with the location of an existing device. However in that case you can create a "reservation" with a comment saying it's the new home of a particular device.
